### ISSUE TITLE
Unable to see Instagram on iphone on sharing image

### ISSUE THREAD

**Issue by Soumitrasp** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#813]
Hello,

I am sharing an image using "SocialSharing-PhoneGap-Plugin".While sharing an image from android phone on instagram in the sharing list instagram shows and shares efficiently.

But same thing as I tried on iphone it didn't show an instagram in the sharing list.[ I already installed instagram on iphone].

Can some one help me as soon as possible on this scenario as it is blocker for us.

**Comment by EddyVerbruggen** [comment]
Did you also search the readme for notes on Instagram?

**Comment by Soumitrasp** [comment]
Yes I actually tried this using  "**canShareVia('instagram'..**" but it didn't work.
Actually I didn't want a separate button for Instagram for sharing to share.

I actually want the same Instagram option in to sharing list on Iphone  as like android device.

**Comment by EddyVerbruggen** [comment]
I meant the urlscheme whitelisting.

**Comment by Soumitrasp** [comment]
I tried this using following way:

**window.plugins.socialsharing.canShareVia('instagram','https://www.google.nl/images/srpr/logo4w.png', null,null);**

Example wise I give the path of url of image whichI want  to share.

**Comment by EddyVerbruggen** [comment]
Let me try again. Did you [edit the plist](https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin/tree/89711c5a84f08b3cfd923f03a5a73d02f2d2293c#4d-whitelisting-on-ios-9)?

**Comment by Soumitrasp** [comment]
I am tried it using Javascript.
So I didn't  gone through plist

**Comment by Soumitrasp** [comment]
Hello Eddy,

Have you find any solution on problem regarding sharing in image on Instagram using iPhone?

**Comment by EddyVerbruggen** [comment]
Anyone else want to try?

**Comment by thielCole** [comment]
I tried this via ionic and am having issues. I added instagram to my plist and to my config.xml (allow navigation) Which solved the issue of canShareWith() failing. But now if I shareWithOptions() it does not allow instagram as an option. 

When I use shareWith('instagram',...)

Then I receive this error message.

```
SLComposeViewController initWithServiceType failed to get extension for identifier instagram
```

This is at the same time that canShareWith('instagram', ...) will return true
